### DISCUSSION TITLE
Install and configure Mailserver (send via external SMTP)

### DISCUSSION THREAD

**Discussion by MajorTomDE** [tteck/Proxmox#1971] [Ideas]
In many cases it is needed, that a LXC machine is able to send emails.

Unfortunately is not easy to setup a Mailserver who can send E-Mails via external SMTP, e.g. Google.

My suggestion is a script/assistent that install all needed packages for an Mailserver (sendmail? Postfix?) and to help to configure it, including sending a testmail.

Maybe this is a idea for a new script?

**Comment by tteck** [comment]
Can Webmin set that up?

![Screenshot 2023-10-27 12 23 07 PM](https://github.com/tteck/Proxmox/assets/21342670/7e76bdaa-2e11-4418-bc12-504c07d40d0f)

**Reply by tteck** [reply]
I've recently conducted a thorough revamp of the Webmin script's code.
